packages/frontapp/src/app/globalUI/Status/index.tsx

```diff
@@ -8,9 +8,29 @@ interface IStyledBookStatusProps {
 }
 
 export const StyledBookStatusWrapper = styled.div<IStyledBookStatusProps>`
+  padding: ${dimensions.xs_2};
+  background: ${(props) => {
+    switch (props.status) {
+      case StatusTypes.FREE:
+        return colors.bg_free;
+      // todo: add separation between own claimed books and others busy (bg_busy)
+      case StatusTypes.BUSY:
+        return colors.bg_own_claim;
+      case StatusTypes.PROLONG:
+        return colors.bg_own_claim;
+      case StatusTypes.OVERDUE:
+        return colors.bg_error;
+      case StatusTypes.REJECTED:
+        return colors.bg_error;
+      case StatusTypes.PENDING:
+        return colors.bg_own_claim;
+      default:
+        return '';
+    }
+  }};
+  border-radius: ${dimensions.xs_2};
   @media (max-width: ${dimensions.phone_width}) {
     width: fit-content;
```

Comment:
Based on the design, `width: fit-content` should be for desktop resolution too

Reply:
moved style out of media query